src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
... but after the simulated event processing is done, you would have to remove the ```isSimulated``` property, otherwise the donor event processing will be done the same way as the original event processing. So, like:

1. Receive native FocusEvent ```nativeFocusEvent```
2. ```nativeFocusEvent.isSimulated = true;``` as part of ```jQuery.event.simulate()```
3. Process the simulated event.
4. Trigger the actual focus event. If we don't ```delete nativeFocusEvent.isSimulated```, then, during the processing of the actual focus event, ```jqueryFocusEvent.originalEvent.isSimulated``` will also be true, because we didn't remove the isSimulated property after the processing of the simulated event.